package.json

```diff
@@ -91,9 +91,12 @@
     "react-native-web": "^0.17.0",
     "react-native-windows": "^0.64.0",
     "react-test-renderer": "17.0.1",
-    "semantic-release": "^17.4.6",
+    "semantic-release": "^19.0.0",
     "typescript": "^4.5.0"
   },
+  "resolutions": {
+    "npm/chalk": "^4.1.2"
```

Comment:
is it dependency of semantic-release?

Reply:
I had this issue in another repo where the latest version of semantic-release bumped npm, which has a dependency on `chalk@*`, and because chalk@5 is ESM only, the pipeline failed because npm can't import it.